test/e2e/autoscale_test.go

```diff
@@ -58,6 +58,10 @@ const (
 	successRateSLO       = 0.999
 )
 
+// Keep it a var so that we can adjust it externally via -ldflags=-X=knative\.dev/serving/test/e2e.autoscaleSleep=<value>"
+// for environments with longer round-trip times
+var autoscaleSleep = "500"
```

Comment:
Why not just a regular flag? I thought with recent changes we should be able to do this, no?
cc @coryrc

Reply:
I thought it would be an unnecessary noise to add a new flag. This is a corner case, IMO.

Reply:
ugh, doing linker flags to patch unit tests just feels icky :-)

Reply:
Exposing an extra flag for just a single test doesn't feel much better to me:) Anyway, I could also just use `const autoscaleSleep = 500` and leave it. It might be a good default for most cases. And if someone complains that the tests still fail we can increase it a little more.

Reply:
That feels better to me :-)

Reply:
Yes possible, agree not necessary.

Reply:
But if you did think it was necessary, put it in test/e2e_flags.go

Reply:
Actually, I just remembered #3892 where I need to modify something that's basically a constant in most environments. We should have some sort of mechanism to do that. Please feel free to make it a constant and I will take on the task of making this a tunable knob.